frontend/app/app.css

```diff
@@ -63,20 +71,29 @@ body {
   padding: 20px 0;
 }
 
+#mainContainer .row{ /* Fix materialize overflowing rows*/
+  margin-right: 0;
+  margin-left: 0;
+}
+#mainContainer h4, h5{
+  color: rgba(117, 59, 189, 1);
+}
+
+
 .clickable:hover {
   cursor: pointer;
 }
 
 .meme-img-small {
-  max-width: 300px;
+  max-width: 250px;
   width: 100%;
-  margin: 1em;
+  padding: 0.7em;
```

Comment:
hmm maybe. i don't remember why i changed that from margin to padding, at least there was too much of it but it could be with margin yes.